Cargo.toml

```diff
@@ -8,5 +8,17 @@ homepage = "https://github.com/xertai/sdac"
 edition = "2021"
 build = "build-cuda-driver.rs"
 
+[lib]
+crate-type = ["cdylib"]
+
 [build-dependencies]
 bindgen = { git = "https://github.com/rust-lang/rust-bindgen", branch="main" }
+
+[dependencies]
+tokio = { version = "1.0", features = ["macros", "net", "rt-multi-thread"] }
+futures = "0.3.25"
+libc = "0.2.139"
+tarpc = {version = "0.31.0", features = ["full"]}
```

Comment:
you can use [`cargo add`](https://doc.rust-lang.org/cargo/commands/cargo-add.html)  to add dependencies with their features. This ensures the formatting of Cargo.toml is correct.